connector_amazon/models/shop.py

```
@@ -0,0 +1,908 @@
+        for binding in order_bindings:
+            picking = binding._get_last_done_picking()
+            if not picking:
+                continue
+            # Only push if tracking is present
+            if not (picking.carrier_id and picking.carrier_tracking_ref):
+                continue
+            try:
+                binding.with_delay().push_shipment()
+            except Exception:
+                # let the job record the error; continue others
+                continue
```

Ruff (0.15.2)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)